tests/reftests/switch-creation.test

```diff
@@ -49,7 +49,7 @@ Done.
 ${BASEDIR}/OPAM/undef-switch
 ### # switch with repo link
 ### # expects a warning, useful to check chosen backend
-### opam switch create w-repo --repositories=from-link=git+https://github.com/ocaml/opam-repository.git --dry-run --empty
+### opam switch create w-repo --repositories=from-link=https://github.com/ocaml/opam-repository.git --dry-run --empty
```

Comment:
The reason we split tests and changes in separate commits is to the what has changed easier.
I think it would be better to not change the tests and show the command fail in the previous commit, then in this one you'd be able to see what the actual change is.